prc/bls/generalities/total_size.tex

```diff
@@ -0,0 +1,15 @@
+The $\blsTotalSize$ varies depending on the precompile (through the value of $\indexMax$), if data or results are considered and whether the call is successful (if it fails, then $\blsTotalSize = 0$). 
```

Comment:
I would rewrite the first sentence to something à la
```
The value of $\blsTotalSize$ depends on several factors.
The first distinction is between the data and result phases.
Along result rows $\blsTotalSize$ is entirely determined by the precompile (and we adopt the convention that $\blsTotalSize \equiv 0$ in case of a failure.)
Along data rows $\blsTotalSize$ is also entirely known, at least for precompiles with fixed input size.
In general one can connect $\blsTotalSize$ to $\indexMax$.
```